src/sentry/options/defaults.py

```diff
@@ -913,9 +913,9 @@
 )
 
 register(
-    "issues.severity.seer-timout",
+    "issues.severity.seer-timeout",
```

Comment:
i checked that this isn't set in options-automator, so should be safe to fix this typo as well

Reply:
checked again